skrub/_gap_encoder.py

```diff
@@ -765,6 +809,12 @@ def _transform(self, X, is_null):
         self.H_dict_ = pre_trans_H_dict_
         return result
 
+    def _post_process(self, X, result):
```

Comment:
If factoring out a method does not lead to reusing it across multiple places, I would rather not factor out the code.

Reply:
We actually use it in `fit_transform` and `transform`